runtime/src/accounts_index.rs

```diff
@@ -1143,7 +1142,7 @@ impl<T: IndexValue> AccountsIndex<T> {
     ) {
         if !dead_keys.is_empty() {
             for key in dead_keys.iter() {
-                let w_index = self.get_account_maps_write_lock(key);
+                let w_index = self.get_account_maps_read_lock(key);
```

Comment:
the `w_` prefix on these variables indicated they were write-locked. Now that we use read-locks instead, we should probably change the prefix to `r_` to avoid confusion.

Reply:
yes, you're right. But, all this code is about to disappear in the next pr.